src/core1_task.rs

```diff
@@ -434,14 +596,21 @@ pub fn core_1_task(
             &mut event_logger,
             &synced_date_time,
             None,
+            false,
         )
     } else {
         false
     };
-    // Unset the is_recording flag on attiny on startup
-    let _ = shared_i2c
-        .set_recording_flag(&mut delay, false)
-        .map_err(|e| error!("Error setting recording flag on attiny: {}", e));
+
+    if record_audio {
+        info!(
+            "Alarm scheduled for {} {}:{}",
+            next_audio_alarm.unwrap().day(),
```

Comment:
It's not super clear from the code that `next_audio_alarm` is always safe to unwrap when `record_audio` is true.  I would prefer to see an `if let Some(next_audio_alarm) = next_audio_alarm { ... }` here.